examples/vllm_inference/flow.py

```diff
@@ -40,69 +41,51 @@ def pull_model_from_huggingface(self):
         )
         self.next(self.run_vllm, num_parallel=2)
 
+    @card
     @model(
+        # this ensures that all workers have the model loaded to the same path
         load=[("llama_model", "./llama_model")]
-    )  # this ensures that all workers have the model loaded to the same path
-    @pypi(
-        python="3.10.11",
-        packages={
-            "vllm": "0.6.1",
-            "transformers": "4.44.2",
-            "huggingface-hub": "0.25.1",
-            "setuptools": "74.1.2",
-        },
     )
     @kubernetes(
         cpu=16,
         gpu=8,
-        memory=MEMORY,
+        memory=60000,
         node_selector="gpu.nvidia.com/class=A100_NVLINK_80GB",
         image="registry.hub.docker.com/valayob/gpu-base-image:0.0.9",
```

Comment:
the `node_selector` can be commented out for the purpose of the Example. You can leave a comment saying that if you are running it on coreweave on the OB platform then uncomment this value.